tests/test_x2pdf.py

```diff
@@ -9,23 +9,42 @@
 from .conftest import run_cli
 
 
-def test_x2pdf(capsys, tmp_path: Path) -> None:
```

Comment:
Could you please preserve the original unit test with no `--format` provided, to ensure it still produces the same PDF document?

Reply:
I see that you preserved the original unit test, thank you, but you also totally removed the tests you intially added 😅

It it is still important to add tests!
I even made a suggestion regarding them: https://github.com/py-pdf/pdfly/pull/65/files/7eab9757363293e76d0571c4aa52ff8db7eb904f#diff-46674680c18f875da306ed40acb72f5a4f0585d0074688e2936fedb48e6a0993